src/pageEditor/fields/UrlMatchPatternField.tsx

```diff
@@ -30,15 +30,15 @@ export type UrlMatchPatternFieldProps = {
 
 const defaultDescription = (
   <span>
-    URL match pattern for which pages to run the extension on. See{" "}
+    URL match patterns for which pages to run the extension on. See{" "}
     <a
       href="https://developer.chrome.com/docs/extensions/mv2/match_patterns/"
       target="_blank"
       rel="noreferrer"
     >
       Patterns Documentation
```

Comment:
Since we're introducing a new type of pattern, we should probably start being specific about what type it expects. Maybe this copy can be improved further:

```suggestion
    URL pattern describing what pages to run the extension on. See the{" "}
    <a
      href="https://developer.chrome.com/docs/extensions/mv2/match_patterns/"
      target="_blank"
      rel="noreferrer"
    >
      `match_patterns` Documentation
```